wgpu/tests/external_texture.rs

```diff
@@ -244,7 +244,7 @@ async fn image_bitmap_import() {
                         dest_layers = 2;
                     }
                     TestCase::SecondSliceCopy => {
-                        correct = false; // TODO: what?
+                        correct = true;
```

Comment:
I think we can remove `correct` altogether now.

Reply:
Done.